pom.xml

```diff
@@ -734,26 +747,24 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>jdk8</id>
+      <activation>
+        <jdk>[1.8,]</jdk>
+      </activation>
+      <properties>
+        <!-- last version working with java 8 -->
+        <spotless-maven-plugin.version>2.30.0</spotless-maven-plugin.version>
```

Comment:
I would like to not mix versions with different JDK, it can happen that different version of spotless will format in different way.

We can set property `recommendedJavaBuildVersion` to 11 to show warning, or even drop this property - we will have a check on CI in the worst case

Reply:
I've been using this for other projects for more than one year and have never had any issues...
jdk8 will be used less and less. Not sure we need to worry about non/false problems..
let;s make this simple...

Reply:
This profile makes it hard to use this parent with JDK21, as for anything matching `[1.8,]` the version 2.43.0 (from main properties) is downgraded back to 2.30.0).
The result is
```
Error:  Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.30.0:check (spotless-check) on project idlj-maven-plugin: Execution spotless-check of goal com.diffplug.spotless:spotless-maven-plugin:2.30.0:check failed: An API incompatibility was encountered while executing com.diffplug.spotless:spotless-maven-plugin:2.30.0:check: java.lang.NoSuchMethodError: 'com.sun.tools.javac.tree.JCTree com.sun.tools.javac.tree.JCTree$JCImport.getQualifiedIdentifier()'
```

Reply:
Given the profile `java11+` also has the range and `+` in id, while `jdk8` is w/o `+` - I guess the `[1.8]` was meant to be here (or `[1.8, 9)`).